src/NserviceBus.Transport.Msmq/MsmqTimeToBeReceivedOverrideCheck.cs

```diff
@@ -19,8 +19,19 @@ public StartupCheckResult CheckTimeToBeReceivedOverrides()
             var isTransactional = settings.GetRequiredTransactionModeForReceives() != TransportTransactionMode.None;
             var outBoxRunning = settings.IsFeatureActive(typeof(Features.Outbox));
 
-            var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
-            var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //var messageAuditingConfig = settings.GetOrDefault<AuditConfigReader.Result>();
+            //var auditTTBROverridden = messageAuditingConfig?.TimeToBeReceived > TimeSpan.Zero;
+
+            //workaround for AuditConfigReader.Result not being accessible
```

Comment:
I'm not sure we want to 🔥 the feature for everyone just because MSMQ can't support it.

At the very least, if we stored the two audit values in settings as separate values with string keys instead of wrapped in the internal `Result` class, the TTBR value would be readable without `dynamic` trickery.